### DISCUSSION TITLE
Is regular auto-pull possible / planned?

### DISCUSSION THREAD

**Discussion by tennox** [gopasspw/gopass#2127] [Ideas]
Hey, I wonder if I always have to `gopass sync` before I get the latest data, or is there a way to auto-pull (and not just before push as the like-named option does)?

**Comment by AnomalRoil** [comment]
Currently you have to do so, yeah. 

We are overall trying to minimize the amount of options we have in our config and also trying to "tidy" our feature set to achieve some kind of "feature completeness", so not sure if we want to add the extra code to handle this, unless there is significant demand from our user-base.

WDYT @dominikschulz ?

**Reply by dominikschulz** [reply]
I like the idea but I'm sure some people would be upset about extra network ops before starting to work.
And even if it's only every x hours it might interrupt batch operations. So I see little chance of us doing this in the short term.
